src/disco/css/App.scss

```diff
@@ -48,37 +42,108 @@ header {
     margin: 0 0 20px;
   }
 
-  p {
+  .content {
     color: $header-copy-font-color;
     font-size: 14px;
     line-height: 1.28;
   }
 
-  p:last-child {
+  .content:last-child {
```

Comment:
I've been trying to avoid nesting. I think I'll change this class name to `disco-content` and remove it from `header` then nest it. I'll also add a class to `header` because that's worrying me.